client/src/redux/settings/settings-sagas.js

```diff
@@ -126,6 +129,18 @@ function* updateMyThemeSaga({ payload: update }) {
   }
 }
 
+function* updateMyKeyboardShortcutsSaga({ payload: update }) {
+  try {
+    const response = yield call(putUpdateMyKeyboardShortcuts, update);
+    yield put(
+      updateMyKeyboardShortcutsComplete({ ...response, payload: update })
+    );
+    yield put(createFlashMessage({ ...response }));
+  } catch (e) {
+    yield put(updateMyKeyboardShortcutsError);
+  }
+}
```

Comment:
```suggestion
function* updateMyKeyboardShortcutsSaga({ payload: update }) {
  try {
    const { data } = yield call(putUpdateMyKeyboardShortcuts, update);
    yield put(updateMyKeyboardShortcutsComplete({ ...data, payload: update }));
    yield put(createFlashMessage({ ...data }));
  } catch (e) {
    yield put(updateMyKeyboardShortcutsError);
  }
}
```
Since https://github.com/freeCodeCamp/freeCodeCamp/pull/46064 has gone in, this needs to pull the data out of the reponse, rather than just using it directly.